package/batocera/core/batocera-configgen/configgen/configgen/generators/mame/mameGenerator.py

```diff
@@ -577,9 +577,9 @@ def generate(self, system, rom, playersControllers, metadata, guns, wheels, game
             mameControllers.generatePadsConfig(cfgPath, playersControllers, messModel, buttonLayout, customCfg, specialController, bezelSet, useGuns, guns, useWheels, wheels, useMouse, multiMouse, system)
 
         # If user provided a custom cmd file at the default location, use that as the customized commandArray
-        if Path(defaultCustomCmdFilepath := f"{rom}.cmd").is_file():
+        if Path(defaultCustomCmdFilepath := f"{rom}.mamecmd").is_file():
             with open(defaultCustomCmdFilepath) as f:
-                commandArray = f.read().splitlines()  # type: ignore
+                commandArray = f.read().strip("\n").splitlines()  # type: ignore
```

Comment:
`splitlines()` splits based on `\n`, so I'm a bit confused as to what this is trying to accomplish